src/pages/workspaces/workspace/tools/FailureRerunner.js

```diff
@@ -46,7 +46,7 @@ export const rerunFailures = async ({ namespace, name, submissionId, configNames
     ])
 
     const failedEntities = _.flow(
-      _.filter({ status: 'Failed' }),
+      _.filter(v => { return (v.status === 'Aborted' || v.status === 'Failed') }),
```

Comment:
nit: since this is a short, single-expression function, it would be clearer without the braces/return.